### PR TITLE
Update to 0.17.0

### PR DESCRIPTION
**Pull request by ocefpaf**
@kain88-de this worked for me locally. Fingers crossed.

### CODE REVIEW

**Comment by kain88-de** [comment]
@ocefpaf thanks. You found the same issue I did. `pip` was missing as a  dependency when we switched from setuptools for the installation.

**Comment by ocefpaf** [comment]
> @ocefpaf thanks. You found the same issue I did. `pip` was missing as a  dependency when we switched from setuptools for the installation.

Note that the safest way to pip install stuff is with `python -m` and `--ignore-installed`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,13 +19,16 @@ requirements:
   build:
     - python
     - pip
-    - numpy 1.10.4
+    - numpy 1.10.4  # [not win]
+    - numpy 1.11.*  # [win]
```
Comment (kain88-de) [inline recipe/meta.yaml]: we really do not support windows for MDAnalysis.Even if the package can be build it won't work correctly.

Comment (ocefpaf) [reply recipe/meta.yaml]: That is not for me to decided, I don't really use `mdanalysis` :grimacing:

Comment (kain88-de) [reply recipe/meta.yaml]: True. But I wouldn't want to give users a broken package. I'm actually not sure why appveyor even tried to build this.

Comment (ocefpaf) [reply recipe/meta.yaml]: B/c I removed the skip statement just for testing. Let me fix that for you.

**Pull request merged**